CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ but technically it is a breaking change to the Elasticsearch mappings.
 - **CUMULUS-2592**
   - Adds logging when messages fail to be added to queue
 
+- **CUMULUS-2577**
+  - Adds `POST` to `executions` endpoint to create an execution
```

Comment:
```suggestion
  - Adds `POST` `/executions` endpoint to create an execution
```

Reply:
We also need to make a PR to the cumulus API docs about this new endpoint

Reply:
PR for cumulus API docs: https://github.com/nasa/cumulus-api/pull/308